android/config.json

```diff
@@ -5,7 +5,8 @@
   "libraries": [
   ],
   "jars": [
-    "libs/KochavaSDK.jar"
+    "libs/tracker-3.4.0.jar",
+    "libs/install-referer-1.0.jar"
```

Comment:
is this needed ?

Reply:
yes. needed https://support.kochava.com/sdk-integration/sdk-kochavatracker-android/migration-guide